### PR TITLE
Update connection-pooling.md

### PR DESCRIPTION
**Pull request by danieltprice**
General edits of the connection pooling topic to improve readability.

### CODE REVIEW

File: content/docs/get-started-with-neon/connection-pooling.md [hunk 1]
```diff
@@ -4,16 +4,24 @@ title: Connection pooling
...
 
-Some apps tend to open many connections and keep most of them inactive. Such behavior usually happens because of database driver limitations or due to the many deployed instances of the app. To deal with such situations, we use a connection pooler.
+Some applications open numerous connections, with most eventually becoming inactive. This behavior can often be attributed to database driver limitations or to running many instances of the application. To handle these situations, Neon supports  connection pooling.
 
 ## Pooling in Neon
 
-Connection pooler is deployed near the postgres and allows to accept up to 500 connections. This connection is routed to the smaller amount of real postgres connections. We use `pgbouncer` in `transaction mode` for connection pooling.
+Enabling connection pooling allows PostgreSQL to accept up to 1000 connections. The connections are routed to a smaller number of real PostgreSQL connections. Neon uses `pgbouncer` in `transaction mode` for connection pooling. For information about `pgbouncer`, refer to <https://www.pgbouncer.org/>.
```
Comment (ololobus) [inline content/docs/get-started-with-neon/connection-pooling.md:13]: Maybe make `<https://www.pgbouncer.org/>` a markdown-formatted link?

File: content/docs/get-started-with-neon/connection-pooling.md [hunk 2]
```diff
@@ -4,16 +4,24 @@ title: Connection pooling
...
-To enable pooling go to the project settings and tick `Enable pooling` checkbox under the `General` section.
+## Enabling connection pooling
+
+To enable connection pooling for a Neon project:
+
+1. Navigate to the [Neon console](https://console.neon.tech/).
+2. On the **Dashboard** tab, select your project from project drop-down list.
+3. Select the **Settings** tab.
+4. Select **General** from the navigation sidebar.
+5. Toggle **Enable pooling** to the on position.
```
Comment (ololobus) [inline content/docs/get-started-with-neon/connection-pooling.md:23]: 6. Click 'Save'? Or it's excessive?

File: content/docs/get-started-with-neon/connection-pooling.md [hunk 3]
```diff
@@ -4,16 +4,24 @@ title: Connection pooling
...
 
 ### Limitations
 
-Some database features like prepared statements and `LISTEN/NOTIFY` functionality won't work with connection pooling. To get a complete list of limitations, see `Transaction pooling` column in the features table at <https://www.pgbouncer.org/features.html>.
+PostgreSQL features such as prepared statements and [LISTEN](https://www.postgresql.org/docs/9.1/sql-listen.html)/[NOTIFY](https://www.postgresql.org/docs/current/sql-notify.html) are not supported with connection pooling. For a complete list of limitations, refer to the "_SQL feature map for pooling modes_" section, in the [pgbouncer.org Features documentation](https://www.pgbouncer.org/features.html).
```
Comment (ololobus) [inline content/docs/get-started-with-neon/connection-pooling.md:27]: Better link the actual documentation version (14 or current), not 9.1

**Review by ololobus — APPROVED** [review]
See some comments. Otherwise, content itself looks good to me

File: content/docs/get-started-with-neon/connection-pooling.md (hunk 1 above)
Comment (danieltprice) [reply content/docs/get-started-with-neon/connection-pooling.md:13]: Fixed!

File: content/docs/get-started-with-neon/connection-pooling.md (hunk 3 above)
Comment (danieltprice) [reply content/docs/get-started-with-neon/connection-pooling.md:27]: Good catch! Fixed.

File: content/docs/get-started-with-neon/connection-pooling.md (hunk 2 above)
Comment (danieltprice) [reply content/docs/get-started-with-neon/connection-pooling.md:23]: Added!

**Comment by danieltprice** [comment]
And thanks Alexey!

**Review by raoufchebri — APPROVED** [review]

**Pull request merged**